internal/resources/fetcher.go

```diff
@@ -29,6 +32,7 @@ type Fetcher struct {
 	// FQDN of the policy server to query. If not empty, Fetcher will query on
 	// port 3000. Useful for out-of-cluster debugging
 	policyServerURL string
+	clientset       kubernetes.Interface
```

Comment:
nit: Maybe worth documenting the struct fields on why we have the dynamic client (for CRDs) and the normal client (for discovery).